include/solvers/unsteady_solver.h

```diff
@@ -204,9 +204,14 @@ class UnsteadySolver : public TimeSolver
 private:
 
   /**
-   * A pointer to a vector to hold the adjoint solution at the last time step
+   * A vector of pointers to vectors holding the adjoint solution at the current time step
    */
-  std::unique_ptr<NumericVector<Number>> old_adjoint;
+  std::vector< NumericVector<Number>* > current_adjoints;
```

Comment:
Do we need an extra variable to hold pointers to the things we're already storing in the System?  Even if it was useful for clarity in the actual method there'd be no reason to keep it in the class rather than as a local variable.

Reply:
Removed.